less/index.less

```diff
@@ -54,6 +54,110 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	font-size: 62.5%;
+	
+	
+}
+body {
+	font-size: 1.6rem;
+	
+	margin: 0 auto;
+	max-width: 1200px;
+}
+
+@resumefontcolor:#6495ED;
+
+.navigation(){
+    // border: 2px solid blue;
+	// background-color: grey;
+	display: flex;
+	justify-content: space-evenly;
+	text-decoration: none;
+	margin-left: 600px;
+	
+}
+
+.navbar{
+	.navigation();
+}
+
+.intro{
+	font-size: 2rem;
+		h1{
+		font-size: 5rem;
+			p{
+			font-size: 1.4rem;
+			}
+		}	
+	}
+
+.theflexervert(){
+	display: flex;
+	flex-direction: column;
+	justify-content: space-evenly;
+	align-items: center;
+	height: 400px;
+}
+
+.top{
+	.theflexervert();
+}
+
+.theflexerhor(){
+	display: flex;
+	
+	justify-content: space-evenly;
+	align-items: center;
+	width:80%
+}
+
+.middle{
+	.theflexerhor();
 }
 
+h2{
+	font-size: 2.5rem;
+	margin-bottom: 5px;
+	color: @resumefontcolor;
+}
+
+h3{
+	font-size: 2.2rem;
+	margin-bottom: 4px;
+	color:@resumefontcolor;
+}
+
+li{
+	margin-left: 5px;
+	margin-top: 3px;
+	margin-bottom: 3px;
+}
+
+footer{
+	margin-left: 500px;
+	margin-top: 30px;
+	color: @resumefontcolor;
+}
+
+
+@media(max-width: 500px){
```

Comment:
good job including the nested media query. Shows you understand the concept and how to execute on it